docs/Glossary/index.md

```diff
@@ -0,0 +1,143 @@
+
+---
+
+## Media Management Applications
+
+**Radarr**, **Sonarr**, **Lidarr**, and **Prowlarr** are applications designed to help with management of your media libraries and automation. This group of applications is commonly referenced collectively as the "arrs".
+
+| Application                          | Content Type       |
+| ------------------------------------ | ------------------ |
+| [**Radarr**](../Radarr/index.md)     | Movies             |
+| [**Sonarr**](../Sonarr/index.md)     | TV Shows/Series    |
+| [**Lidarr**](../Lidarr/index.md)     | Music              |
+| [**Prowlarr**](../Prowlarr/index.md) | Indexer Management |
+
+### Custom Format
+
+A feature that uses pattern matching on a release, based on the information implied through the release name on the corresponding indexer during a search. Users can customize patterns to match a Custom Format, and Custom Formats can then be assigned scores individually in each [Quality Profile](#quality-profile). After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.
+
+### Quality
+
+Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.
+
+### Quality Definitions
+
+Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.
+
+### Quality Profile
+
+A defined group and preferential ordering of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.
+
+---
+
+## Torrents
+
+### Cross-Seed
+
+The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org) that automates this process. `Cross-Seed` and `crossseed` or `xseed` often refer to the application, while `cross seed` or `xseed` can refer to the action.
+
+### Deluge
+
+[Deluge](https://deluge-torrent.org/) is a lightweight, free, open-source, cross-platform BitTorrent client.
+
+### qBit Manage
+
+[qBit Manage](https://github.com/StuffAnThings/qbit_manage) (aka `qbm` or `qbitmanage` / `qb_manage`)  program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes such as time and tracker.
+
+### qBittorrent
+
+[qBittorrent ](https://www.qbittorrent.org/) (aka `qb`, `qbit`, `qbittorrent`) is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions until properly validated by the community, as these often contain significant changes and/or bugs.
```

Comment:
```suggestion
[qBittorrent](https://www.qbittorrent.org/) (aka `qb`, `qbit`, `qbittorrent`) is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions until properly validated by the community, as these often contain significant changes and/or bugs.
```
lint issue.

Reply:
Resolved in latest push.